jest.config.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+  moduleFileExtensions: ['js', 'jsx', 'json', 'vue', 'ts', 'tsx'],
+  transform: {
+    '^.+\\.vue$': 'vue-jest',
+    '.+\\.(css|styl|less|sass|scss|svg|png|jpg|ttf|woff|woff2)$':
+      'jest-transform-stub',
+    '^.+\\.tsx?$': 'ts-jest',
+  },
+  transformIgnorePatterns: ['/node_modules/'],
+  moduleNameMapper: {
+    '^@/(.*)$': '<rootDir>/src/$1',
+  },
+  snapshotSerializers: ['jest-serializer-vue'],
+  testMatch: [
+    '**/tests/unit/**/*.spec.(js|jsx|ts|tsx)|**/__tests__/*.(js|jsx|ts|tsx)|**/*.test.ts',
```

Comment:
by default, jest appears to want test files to be placed in /tests/unit/.../*.spec.ts or __tests__ subdirectories. i prefer storing them alongside the files being tested as *.test.ts. if that sounds reasonable, i should probably just delete the other patterns here...

Reply:
Storing them alongside the files being tested (as *.test.ts) sounds great. I prefer the data files next to the test files, rather than the data files in a separate directory.